proto/server/datastore/interface.go

```diff
@@ -46,6 +46,7 @@ type DataStore interface {
 
 	ListParentIDEntries(request *ListParentIDEntriesRequest) (*ListParentIDEntriesResponse, error)
 	ListSelectorEntries(request *ListSelectorEntriesRequest) (*ListSelectorEntriesResponse, error)
+	ListPowerSelectorEntries(request *ListSelectorEntriesRequest) (*ListSelectorEntriesResponse, error)
```

Comment:
This is a hard one, naming-wise... `ListPowerSelectorEntries` is logical given `ListSelectorEntries`. Another option could be `ListMatchingEntries` or even `FindMatchingEntries`.  :shrug:

Reply:
I might go for ListMatchingEntries since I don't like ListPowerSelectorEntries.